packages/migration/src/migrations/test/v0028_video3p.js

```diff
@@ -0,0 +1,231 @@
+                  lengthFormatted: '1:01:32',
+                  length: 3692,
+                  mimeType: 'video/mp4',
+                  width: 365,
+                  height: 627,
+                },
+                x: 129,
+                y: 155,
+                width: 265,
+                height: 527,
+                fontSize: 58,
+              },
+            ],
+          },
+        ],
+      })
+    ).toStrictEqual({
+      _test: 'story',
+      pages: [
+        {
+          _test: 'page1',
+          elements: [
+            {
+              _test: 'element1',
+              type: 'square',
+              x: 10,
+              y: 20,
+              width: 100,
+              height: 200,
+            },
+            {
+              _test: 'element2',
+              type: 'image',
+              resource: {
+                type: 'image',
+                mimeType: 'image/png',
+                src: 'https://example.com/image.png',
+                width: 265,
+                height: 527,
+              },
+              x: 29,
+              y: 55,
+              width: 265,
+              height: 527,
+              fontSize: 58,
+            },
+          ],
+        },
+        {
+          _test: 'page2',
+          elements: [
+            {
+              _test: 'element3',
+              type: 'video',
+              resource: {
+                type: 'video',
+                src: 'https://example.com/video.mp4',
+                poster: 'https://example.com/image.png',
+                posterId: 100002,
+                id: 102113,
+                lengthFormatted: '0:32',
+                length: 32,
+                mimeType: 'video/mp4',
+                width: 265,
+                height: 527,
+              },
+              x: 29,
+              y: 55,
+              width: 265,
+              height: 527,
+              fontSize: 58,
+            },
+            {
+              _test: 'element4',
+              type: 'video',
+              resource: {
+                type: 'video',
+                src: 'https://example.com/video1.mp4',
+                poster: 'https://example.com/image1.png',
+                posterId: 100003,
+                id: 'media/coverr:aSb45qtWTus',
+                isOptimized: true,
+                lengthFormatted: '1:32',
+                length: 92,
+                mimeType: 'video/mp4',
+                width: 365,
+                height: 627,
+              },
+              x: 129,
+              y: 155,
+              width: 265,
+              height: 527,
+              fontSize: 58,
+            },
+            {
+              _test: 'element5',
+              type: 'video',
+              resource: {
+                type: 'video',
+                src: 'https://example.com/video2.mp4',
+                poster: 'https://example.com/image2.png',
+                posterId: 100006,
+                id: 'media/cover:aSb45qtWTus',
```

Comment:
Let's use a Tenor ID or something like simply `12345` for this case. `media/cover:aSb45qtWTus` is not something that would exist "in the wild"